mmdeploy/codebase/mmcls/models/backbones/vision_transformer.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch import Tensor
+from torch.nn import Linear
+from mmcls.models.utils import channel_shuffle, resize_pos_embed
+
+from mmdeploy.core import FUNCTION_REWRITER
+from mmdeploy.utils import Backend
+from typing import Tuple
+
+
+# ncnn have implemented MultiheadAttention, onnx would split this opr. So the model needs to rewrite.

```

Comment:
Comments about why we need this rewrite can be placed in the docstring of this function.

Reply:
done.